examples/lighting-app/nrf5/BUILD.gn

```diff
@@ -14,6 +14,7 @@
 
 import("//build_overrides/chip.gni")
 import("//build_overrides/nrf5_sdk.gni")
+import("//build_overrides/openthread.gni")
```

Comment:
is this part relevant to BuildConfig.h?

Reply:
This is unrelated I just noticed it was missing from lighting app (it's in lock-app).